src/Symfony/Bundle/FrameworkBundle/Resources/config/messenger.xml

```diff
@@ -43,12 +43,14 @@
         </service>
 
         <!-- Message encoding/decoding -->
-        <service id="messenger.transport.serialize_message_with_type_in_headers" class="Symfony\Component\Messenger\Transport\Serialization\Serializer">
+        <service id="messenger.transport.symfony_serializer" class="Symfony\Component\Messenger\Transport\Serialization\Serializer">
             <argument type="service" id="serializer" />
+            <argument /> <!-- Format -->
+            <argument type="collection" /> <!-- Context -->
         </service>
 
-        <service id="messenger.transport.default_encoder" alias="messenger.transport.serialize_message_with_type_in_headers" public="true" />
-        <service id="messenger.transport.default_decoder" alias="messenger.transport.serialize_message_with_type_in_headers" public="true" />
+        <service id="messenger.transport.symfony_serializer_encoder" alias="messenger.transport.symfony_serializer" />
```

Comment:
Are those aliases really needed? I think `messenger.transport.symfony_serializer` is fine as default `encoder`/`decoder` config value.

Reply:
Yeah, definitely now that we have the aliases generated by FWB.